target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -218,6 +218,10 @@ qxwlan,e750a-v4-16m)
 	ucidef_set_led_switch "lan" "LAN" "green:lan" "switch0" "0x02"
 	ucidef_set_led_netdev "wan" "WAN" "green:wan" "eth1"
 	;;
+glinet,gl-x300b|\
+	ucidef_set_led_netdev "wan"   "WAN"   "green:wan" "eth0"
```

Comment:
Either this is wrong or your setup in 02_network is.

Reply:
It was wrong, it's been corrected.